projects/cashmere/src/lib/modal/modal.service.ts

```diff
@@ -19,9 +19,14 @@ export type ModalContentType = Type<{}> | TemplateRef<any>;
 
 @Injectable()
 export class ModalService {
+
+    /** Restricts multiple modals from being opened on top of each other (an error will be thrown if attempted). */
+    allowMultiple: boolean = false;
```

Comment:
This is a BREAKING CHANGE unless you default it to `true`.

Reply:
As the commit is not flagged as breaking change, I would set this to `true` as default.